pkgs/t3/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  help2man,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "t3";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "flox";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-NB1aNnMnFhv5rvTvml7vLSJlH94OW9zplttk8HrAy+8=";
+  };
+
+  installFlags = [ "PREFIX=$(out)" ];
+  nativeBuildInputs = [ help2man ];
+  doCheck = true;
+
+  meta = with lib; {
+    homepage = "https://github.com/flox/t3";
+    description = "Next generation tee with colorized output streams and precise time stamping";
+    maintainers = [ maintainers.limeytexan ];
+    license = licenses.mit;
+  };
+}
```

Comment:
@ysndr what was the concern you had about how adding a log output would affect rebuilding packages?

Reply:
Yannik had observed that Nix expression for packaging impure builds listed the logfile as an input, and therefore would not be reproducible. I've since updated the logic to add the `log` package as a separate derivation so that's no longer an issue.

And just for completeness, the `log` package continues to be an output of the pure builds because that's the only way to capture and extract the log from the build sandbox (kinda like `buildCache`). It's not a problem for this `log` output to be built in different ways because our ultimate aim is simply to upload it to a binary cache and list its path in the catalog, just like any other package.